radio/src/gui/horus/menu_model_mixes.cpp

```diff
@@ -523,12 +555,13 @@ bool menuModelMixAll(evt_t event)
 
           displayMixLine(y, md);
 
-          char cs[] = " ";
+          BitmapBuffer * delayslow = 0;
           if (md->speedDown || md->speedUp)
-            cs[0] = 'S';
+            delayslow = mixerSetupSlowBitmap;
           if (md->delayUp || md->delayDown)
-            cs[0] = (cs[0] =='S' ? '*' : 'D');
-          lcdDrawText(MIX_LINE_DELAY_POS, y, cs);
+            delayslow = (delayslow ? mixerSetupDelaySlowBitmap : mixerSetupDelayBitmap );
+          if (delayslow)
+            lcd->drawBitmap(MIX_LINE_DELAY_SLOW_POS, y + 2, delayslow);
 
```

Comment:
to avoid one if: an array of 3 bitmaps, delayslow initialized at 0, speed => +=1 delay => +=2 
if delayslow draw bitmap(delayslow-1)